### PR TITLE
feat: add new line for consecutive incert uses

### PR DESCRIPTION
**Pull request by pedrotiag0**
Hello, great tool!

When I need to use _incert_ consecutive times in the same image, the `ca-certificates.crt` file gets unformatted.

Expected behavior:
```
-----BEGIN CERTIFICATE-----
ABC1
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
ABC2
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
ABC3
-----END CERTIFICATE-----
```

Actual behavior:
```
-----BEGIN CERTIFICATE-----
ABC1
-----END CERTIFICATE----------BEGIN CERTIFICATE-----
ABC2
-----END CERTIFICATE----------BEGIN CERTIFICATE-----
ABC3
-----END CERTIFICATE-----
```

This pr aims to add a new line after appending the certificate (in order to achieve the expected behavior).

### CODE REVIEW

**Comment by vipulagarwal** [comment]
Hi @pedrotiag0, thanks for fixing this issue. Could you sign your commits please?
ref: https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

You can use `gitsign` and make use of public Sigstore to sign your commits.
https://docs.sigstore.dev/signing/gitsign/

**Comment by pedrotiag0** [comment]
Sorry for the inconvenience @vipulagarwal . I signed my commit following GitHub's documentation :) Let me know if I should be doing it through sigstore (to fix pipeline).

**Comment by vipulagarwal** [comment]
Thanks @pedrotiag0. I will wait for the repo maintainers to review and comment.

**Comment by amouat** [comment]
Interesting, thanks. I wonder if the append would be cleaner on a different line, but it's not worth worrying about if this works.

**Review by amouat — APPROVED** [review]

**Pull request merged**
